src/vm/i386/excepx86.cpp

```diff
@@ -38,6 +38,7 @@
 #include "asmconstants.h"
 #include "virtualcallstub.h"
 
+#ifndef WIN64EXCEPTIONS
```

Comment:
@parjong I just tried to ifdef-out what I've said and have built it on a x86 Linux. Then used `ldd -r libcoreclr.so` to see the missing symbols. There were just two:
```
AdjustContextForVirtualStub(_EXCEPTION_RECORD*, _CONTEXT*)
CLRNoCatchHandler(_EXCEPTION_POINTERS*, void*)
```